mne/io/meas_info.py

```diff
@@ -1577,6 +1577,10 @@ def anonymize_info(info):
         del info['subject_info']
     info['meas_date'] = [0, 0]
     for key_1 in ('file_id', 'meas_id'):
+        if (key_1 not in info.keys()) or (info[key_1] is None):
+            continue
         for key_2 in ('secs', 'msecs', 'usecs'):
+            if key_2 not in info[key_1].keys():
```

Comment:
@kingjr this is evil. You make a list for nothing and testing if key present becomes O(n) operation instead of O(1).